src/ripple/ledger/View.h

```diff
@@ -340,8 +340,8 @@ transferXRP (ApplyView& view,
             STAmount const& amount,
                 beast::Journal j);
 
-NetClock::time_point const& flowV2SoTime ();
-bool flowV2Switchover (NetClock::time_point const closeTime);
+NetClock::time_point const& numAccSoTime ();
```

Comment:
`numAcc` is a little too abbreviated for me to make sense of.  Could there be a comment explaining what `numAcc` means?